docs/nulecule.md

```diff
@@ -0,0 +1,242 @@
+# Nulecule file
+
+A lot of information here is copied from the [Nulecule specification](https://github.com/projectatomic/nulecule/tree/master/spec).
+
+There are __minor__ differences as we implement features not yet implemented into `atomicapp` from the specification. Please refer to this reference for deploying your application.
```

Comment:
Just thinking about this, but I don't think this should be added as it may imply that atomicapp's implementation is unstable.